static/src/javascripts/projects/common/modules/save-for-later.js

```diff
@@ -360,7 +360,11 @@ define([
 
             var profile = $('.brand-bar__item--profile');
             profile.addClass('brand-bar__item--profile--show-saved');
-            $('.control__icon-wrapper', profile).attr('data-saved-content-count', count);
+            if (count > 0) {
+                $('.control__icon-wrapper', profile).attr('data-saved-content-count', count);
+            } else {
+                $('.control__icon-wrapper', profile).removeAttr('data-saved-content-count');
```

Comment:
This leads to:

![image](https://cloud.githubusercontent.com/assets/921609/8255660/77724f1c-1699-11e5-80e1-b30f55102b77.png)

I think we need to make the line above only run if the count `> 0`.